### ISSUE TITLE
[Feature Request] allow using other installation services

### ISSUE THREAD

**Issue by barackenkumpel** [NeoApplications/Neo-Store#232]
Foxy Droid is allowing it:
![Screenshot_20220618-143035_Android System](https://user-images.githubusercontent.com/93818968/174437983-f0f20f4b-7db8-4f6a-91e8-f06a44de156f.jpg)

**Comment by barackenkumpel** [comment]
Also, look at how Aurora Store let you choose installation method:
![Screenshot_20220622-072456_Aurora Store.jpg](https://user-images.githubusercontent.com/93818968/174950383-a5a25f2d-e54a-41e7-b65e-db163c0e7a77.jpg)

**Comment by barackenkumpel** [comment]
There is even no need of detecting installation methods. There could be just one other option in the settings which isn't session or root installer, but performs something like following on install:

(copy-pasted from Foxy Droid Source Code)
```
context.startActivity(Intent(context, MainActivity::class.java)
            .setAction(MainActivity.ACTION_INSTALL).setData(Uri.parse("package:$packageName")) [...] )
```

**Comment by machiav3lli** [comment]
Fixed on 1.0.0
